app/utils/sweats_message_handler.js

```diff
@@ -0,0 +1,141 @@
+import { DiscordRequest } from './discord_request.js';
+import { EventEmitter } from 'events';
+const green=0x66FF33
+const red=0xff6633
+
+export async function sweatsMessageHandler() {
+    try {
+        const channelId = process.env.CHANNEL_ID_CSWEATS;
+        const messageId = await sendMessage(channelId);
+        const intervalId = listenForReactions(channelId, messageId);
+        if (intervalId === null) {
+            return;
+        }
+        setTimeout(() => clearInterval(intervalId), 1000 * 60 * 60 * 24); // 24 hours
+    } catch (error) {
+        console.error(error);
+        throw new Error('Error handling sweats message');
+    }
+}
+
+
+function listenForReactions(channelId, messageId) {
+    const endpoint = `channels/${channelId}/messages/${messageId}/reactions/%F0%9F%92%A7`;
+    const listener = new EventEmitter();
+    let previousUsers = [];
+    try {
+        const reactioncheck = setInterval(async () => {
+            var response;
+            try {
+            response = await DiscordRequest(endpoint, {
+                method: 'GET',
+                });
+            }catch (error) {
+                console.log(error);
+                const errorObj = JSON.parse(error.message);
+                if (errorObj.code === 10008) {
+                    clearInterval(reactioncheck);
+                    console.error('Message not found, stopping reaction check');
+                    return null;
+                } else {
+                    console.log(error);
+                    return reactioncheck;
+                }
+            }
+            const users = await response.json();
+            const userIds = users.map(user => user.id);
+
+            if (JSON.stringify(userIds) !== JSON.stringify(previousUsers)) {
+                previousUsers = userIds;
+                listener.emit('reaction', users);
+            }
+        }, 5000); // Check every 5 seconds
+        
+
+        listener.on('reaction', async (users) => {
+            users = users.filter(user => !user.bot);
+            if (users.length != 0) {
+                removeOwnReaction(channelId, messageId);
+            } else {
+                addOwnReaction(channelId, messageId);
+            }
+            editMessage(channelId, messageId, users);
+        });
+
+        return reactioncheck;
+    } catch (error) {
+        console.error(error);
+    }
+    return null;
+}
+
+
+export async function sendMessage(channelId) {
```

Comment:
(Suggestion: architecture) This function is exported, we can either use it to replace the manual message sending we're doing in `command_handler.js` or make this private. I think the former is a good idea, it'd let us add additional parameters like "message", etc. to make this function more generic and able to be used elsewhere